src/tctrack/utils/netcdf.py

```diff
@@ -0,0 +1,12 @@
+"""Module providing utility functions for working with netcdf files."""
+
+from netCDF4 import Dataset
+
+
+def lon_lat_sizes(filename: str, lon: str = "lon", lat: str = "lat") -> tuple[int, int]:
```

Comment:
```suggestion
def lat_lon_sizes(filename: str, lon: str = "lon", lat: str = "lat") -> tuple[int, int]:
```

"lat-lon" is the common parlance in the domain, so I'd rename this to match, and head off common user syntax errors.

Reply:
Thanks!